src/deps_hack/src/lib.rs

```diff
@@ -107,6 +107,9 @@ pub struct ZookeeperConfig {
 #[kube(shortname = "rbmq", namespaced)]
 pub struct RabbitmqClusterSpec {
     pub replicas: i32,
+    /// Image is the name of the RabbitMQ docker image to use for RabbitMQ nodes in the RabbitmqCluster.
+    /// Must be provided together with ImagePullSecrets in order to use an image in a private registry.
+    pub image: String,
```

Comment:
Did you copy the description from the official rabbitmq controller? We don't have an ImagePullSecrets here.